### ISSUE TITLE
Reply to an email missing in conversation

### ISSUE THREAD

**Issue by boehmgui** [freescout-helpdesk/freescout#2818]
Hi team,

we emailed a customer from a 3rd party email client. The 'From' address is the one of our freescout inboxes. The customer's reply hit that inbox, and freescout fetched that email. But for some strange reason, the conversation only shows the text of the email we sent. The actual reply is completely missing. It is even not visible when clicking on "show original". 

The log does not reveal anything strange:
8 | [2023-03-10 16:34:01] Thread successfully created: 872
7 | [2023-03-10 16:34:01] Message from: Customer
6 | [2023-03-10 16:34:01] 1) AW:  Stickvorschlag zu Ihrem Angebot/ Bestellung AU-xxxx-xxxxx-WS
5 | [2023-03-10 16:34:01] Fetched: 1
4 | [2023-03-10 16:34:01] Folder: INBOX

What additional shall I provide for investigation? I could share that email if needed.

freescout version: 1.8.67

**Comment by freescout-helpdesk** [comment]
> we emailed a customer from a 3rd party email client. The 'From' address is the one of our freescout inboxes. The customer's reply hit that inbox, and freescout fetched that email.

The explanation is not quite clear.

Do you see the reply if you log in to the email service to which the FreeScout is connected.

**Comment by boehmgui** [comment]
Let me rephrase..

We used a CRM sytem to send an email to the customer. The CRM sets the FROM address to info@mydomain.com (it is sent through this account).
Customer replied to that email, which hit the inbox of info@mydomain.de
Freescout fetched the email from the inbox of info@mydomain.de.
The conversation in frerscout only shows the content of the email that was sent from CRM. The reply is completely missing.
When I checked the email directly in the inbox of info@mydomain.de I can see the complete content, i.e. initial text _and_ reply

**Comment by freescout-helpdesk** [comment]
Can you give a screenshot of what part is missing.

Also see https://github.com/freescout-helpdesk/freescout/wiki/FAQ#why-does-show-original-window-shows-truncated-message-without-previous-history

**Comment by boehmgui** [comment]
Ok, we seem to have two independent issues here:
1. incomplete message history in "show original": this is because of the module "IMAP move/remove". After fetching, emails are moved to a subfolder. If I move it back into the inbox, at least "show original" shows a complete message. This should be taken into account and I will open a separate issue for it.
2. the issue that the customer's reply is not shown in the conversation as such...

This is the screenshot from the email server's inbox. It shows the initial email to the customer and their reply:

![image](https://user-images.githubusercontent.com/102216227/224633093-5f6b057a-3839-435a-9f7b-99214e72337d.png)

And this is how it is represented in freescout. Only the content of the initial email is visible :

![image](https://user-images.githubusercontent.com/102216227/224633512-e7f614a4-2422-4c87-b24c-2d6d8a70056b.png)

This is the source code of the reply, which I copied from Thunderbird - not sure, whether this is of any help....:
`Return-Path: <customer@some-domain.de>
X-Spam-Checker-Version: SpamAssassin 3.4.4 (2020-01-24) on
	server.mydomain.de
X-Spam-Level: 
X-Spam-Status: No, score=-2.6 required=5.0 tests=BAYES_00,FREEMAIL_FROM,
	HTML_MESSAGE,RCVD_IN_DNSWL_LOW,RCVD_IN_MSPIKE_H3,RCVD_IN_MSPIKE_WL,
	SPF_HELO_NONE,SPF_NONE autolearn=ham autolearn_force=no version=3.4.4
X-Original-To: info@mydomain.de
Delivered-To: info@mydomain.de
Received: from mailout05.t-online.de (mailout05.t-online.de [194.25.134.82])
	by server.mydomain.de (Postfix) with ESMTPS id B1EDF5C1367
	for <info@mydomain.de>; Fri, 10 Mar 2023 13:22:33 +0100 (CET)
Authentication-Results: server.mydomain.de;
	dmarc=none (p=NONE sp=NONE) smtp.from=t-online.de header.from=t-online.de;
	spf=pass (sender IP is 194.25.134.82) smtp.mailfrom=customer@some-domain.de smtp.helo=mailout05.t-online.de
Received-SPF: pass (server.mydomain.de: domain of t-online.de designates 194.25.134.82 as permitted sender) client-ip=194.25.134.82; envelope-from=customer@some-domain.de; helo=mailout05.t-online.de;
Received: from fwd73.dcpf.telekom.de (fwd73.aul.t-online.de [10.223.144.99])
	by mailout05.t-online.de (Postfix) with SMTP id 1EFB41F962
	for <info@mydomain.de>; Fri, 10 Mar 2023 13:22:12 +0100 (CET)
Received: from spica40.aul.t-online.de ([172.20.102.136]) by fwd73.aul.t-online.de
	with esmtp id 1pabkG-1atHUG0; Fri, 10 Mar 2023 13:21:12 +0100
Received: from 217.255.151.156:58720 by spica40.aul.t-online.de:8080; Fri, 10 Mar 2023 13:21:11 +0100 (CET)
Date: Fri, 10 Mar 2023 13:21:11 +0100 (CET)
From: "customer@some-domain.de" <customer@some-domain.de>
Sender: "customer@some-domain.de" <customer@some-domain.de>
To: "info@mydomain.de" <info@mydomain.de>
Message-ID: <1678450871911.178800.a23151fe399975a8ba3de5105f18ac673b30cd33@spica.telekom.de>
In-Reply-To: <f246e13a-7f3a-4d67-b78f-59253fbdf2b6@server.mydomain.de>
References: <f246e13a-7f3a-4d67-b78f-59253fbdf2b6@server.mydomain.de>
Subject: AW:  Stickvorschlag zu Ihrem Angebot/ Bestellung AU-2023-2867-WS
MIME-Version: 1.0
Content-Type: multipart/alternative;
	boundary="----=_Part_227250_1032895432.1678450872043"
X-TOI-EXPURGATEID: 150726::1678450872-B639BDF7-6D20F936/0/0 CLEAN NORMAL
X-TOI-MSGID: edde1430-0aa3-4021-b683-2af076b50c3d

------=_Part_227250_1032895432.1678450872043
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: quoted-printable

Hallo Frau xxxxxx,

da verlasse ich mich ganz auf Sie,aber es gef=C3=A4llt mir sehr gut!
Vielen Dank f=C3=BCr Ihre schnelle R=C3=BCckmeldung.

Sch=C3=B6nes Wochenende

Gru=C3=9F xxxx xxxxx


------------------------------------------------------------------------
Gesendet mit der Telekom Mail App
<https://kommunikationsdienste.t-online.de/redirects/email_app_android_send=
mail_footer>



--- Original-Nachricht ---
Von: info@mydomain.de
Betreff: Stickvorschlag zu Ihrem Angebot/ Bestellung AU-2023-2867-WS
Datum: 10. M=C3=A4rz 2023, 9:34
An: customer@some-domain.de




Guten Tag, xxxxxx xxxxxx,

Vielen Dank f=C3=BCr Ihre Anfrage, bzw. Bestellung.

Anbei sende ich Ihnen zwei Farbvorschl=C3=A4ge mit der Bitte um Freigabe .=
=20
Leider kommen die Farben nicht so sch=C3=B6n raus, aber der Farbton entspri=
cht=20
fast der gro=C3=9Fen Fl=C3=A4che.
Geben Sie mir kurz Bescheid ob es Ihnen so gef=C3=A4llt, ansonsten=20
ber=C3=BCcksichtige ich nat=C3=BCrlich gerne noch
Ihre =C3=84nderungsw=C3=BCnsche.

F=C3=BCr weitere Fragen stehe ich Ihnen nat=C3=BCrlich gerne zur Verf=C3=BC=
gung.

Mit freundlichen Gr=C3=BC=C3=9Fen,

xxxxxx xxxxxxx

xxxxx xxxxxx
xxxxxx xxxxxx
xxxxxxxxxx 9
xxxxxxxx yyyyyyyyyyyyy
Tel.: xxxxxxx/ xxxxxxxxxx
Email: info@mydomain.de <mailto:info@mydomain.de>
WWW:<http://www.mydomain.de>

USt-ID: xxxxxxxxxxx
Steuer-Nr. xxxxxxxxxxxxxxxxxx


Socialmedia
Youtube:<http://www.youtube.com/user/xxxxxxxn>
Facebook:<http://www.facebook.com/xxxxxxxxxxx>

------=_Part_227250_1032895432.1678450872043
Content-Type: text/html; charset="utf-8"
Content-Transfer-Encoding: quoted-printable

Hallo Frau xxxxxx,<br/><br/>da verlasse ich mich ganz auf Sie,aber es ge=
f=C3=A4llt mir sehr gut!<br/>Vielen Dank f=C3=BCr Ihre schnelle R=C3=BCckme=
ldung. <br/><br/>Sch=C3=B6nes Wochenende <br/><br/>Gru=C3=9F xxxxxx xxxxxx <b=
r/><br/><br/><hr></hr>Gesendet mit der <a href=3D"https://kommunikationsdie=
nste.t-online.de/redirects/email_app_android_sendmail_footer">Telekom Mail =
App</a><br> <br><br/><br/>--- Original-Nachricht ---<br/><b>Von:  </b>info@=
mydomain.de<br/><b>Betreff:  </b>Stickvorschlag zu Ihrem Angebot/=
 Bestellung AU-2023-2867-WS<br/><b>Datum:  </b>10. M=C3=A4rz 2023, 9:34 <br=
/><b>An:  </b>customer@some-domain.de<br/><br/><br/><br> <br><html><head>  =
  <meta name=3D"viewport" content=3D"width=3Ddevice-width, initial-scale=3D=
1.0" />    <style type=3D"text/css">@font-face {=09font-family: telegrotesk=
-medium_normal;=09src: url("file:///android_asset/fonts/telegrotesk_normal.=
ttf");}html,body {=09font-family: "telegrotesk-medium_normal";=09font-size:=
 medium;=09color: #4b4b4b;=09width: 100%;}</style></head><body><div>    Gut=
en Tag, xxxxxx xxxxxx,<br/><br/>Vielen Dank f=C3=BCr Ihre Anfrage, bzw. Beste=
llung.<br/><br/>Anbei sende ich Ihnen zwei Farbvorschl=C3=A4ge mit der Bitt=
e um Freigabe . Leider kommen die Farben nicht so sch=C3=B6n raus, aber der=
 Farbton entspricht fast der gro=C3=9Fen Fl=C3=A4che.<br/>Geben Sie mir kur=
z Bescheid ob es Ihnen so gef=C3=A4llt, ansonsten ber=C3=BCcksichtige ich n=
at=C3=BCrlich gerne noch<br/>Ihre =C3=84nderungsw=C3=BCnsche.<br/><br/>F=C3=
=BCr weitere Fragen stehe ich Ihnen nat=C3=BCrlich gerne zur Verf=C3=BCgung=
.<br/><br/>Mit freundlichen Gr=C3=BC=C3=9Fen,<br/><br/>Suxxxxxx e xxxxxx<br=
/><br/>xxxxxx xxxxxx<br/>Suxxxxxx e xxxxxx<br/>Feldbergstrasse 9<br/>6=
1206 xxxxxx<br/>Tel.: xxxx/ xxxxxx<br/>Email: <a href=3D"mailto:=
info@mydomain.de">info@mydomain.de</a><br/>WWW: <a href=
=3D"http://www.mydomain.de" target=3D"_top" >www.xxxxxx es-stickdesi=
gn.de</a><br/><br/>USt-ID: xxxxxx <br/>Steuer-Nr. xxxxxx <br/><br/=
><br/>Socialmedia<br/>Youtube:  <a href=3D"http://www.youtube.com/user/xxxxxxxxx" target=3D"_top" >http://www.youtube.com/user/xxxxxxxxx</a><br/>Facebook: <a href=3D"http://www.facebook.com/xxxxxxxxx"=
 target=3D"_top" >http://www.facebook.com/xxxxxxxxxx</a><br/></div>=
</body></html>
------=_Part_227250_1032895432.1678450872043--

`

**Comment by freescout-helpdesk** [comment]
https://github.com/freescout-helpdesk/freescout/wiki/FAQ#is-it-possible-to-forward-customers-email-to-freescout-and-make-it-look-like-it-came-from-this-customer

**Comment by boehmgui** [comment]
I don't think this is the case. The email was not forwarded by anyone. It came directly from the customer

**Comment by freescout-helpdesk** [comment]
> 1. incomplete message history in "show original": this is because of the module "IMAP move/remove". After fetching, emails are moved to a subfolder. If I move it back into the inbox, at least "show original" shows a complete message. This should be taken into account and I will open a separate issue for it.

This has been fixed. Does Show Original show the email now?

**Comment by freescout-helpdesk** [comment]
Unfortunately we can't reproduce the issue. Try to fix debug it yourself or find someone who can do it on your server: https://github.com/freescout-helpdesk/freescout/wiki/Hire-Developer
